spec/models/user_spec.rb

```diff
@@ -135,6 +135,53 @@
     end
   end
 
+  describe '#validations' do
+    context 'password' do
+      let(:user) { build :user }
+
+      context 'when password is blank' do
+        before do
+          user.save
```

Comment:
do we need to save the user here? we should be able to validate the user before persisting